backend/lib/cache/index.js

```diff
@@ -29,6 +30,11 @@ class Cache {
     this.journalCache = createJournalCache()
   }
 
+  /*
```

Comment:
Synchronize should be async. We will wait for all stores to be synchronized before we start the server listinging to request. If sync fails we do not start server.